src/robot/model/control.py

```diff
@@ -242,3 +242,29 @@ def __init__(self, values=(), parent=None):
 
     def visit(self, visitor):
         visitor.visit_return(self)
+
+
+@Body.register
+class Continue(BodyItem):  
+    type = BodyItem.CONTINUE
+    __slots__ = []
+
+    def __init__(self, parent=None):
+        self.parent = parent
+        #self.body = None
```

Comment:
Remove the commented code here and in `Break`. These structures don't have body.